### PR TITLE
SALTO-2699 Clean Okta nacls

### PR DESCRIPTION
**Pull request by shir-reifenberg**
Change config to clean okta nacls

---

_Additional context for reviewer_
main changes:
- Add fields to `fieldsToOmit` and `fieldsToHide`
- Extract `PolicyRule` to `standaloneFields`

---

### CODE REVIEW

File: packages/okta-adapter/src/config.ts [hunk 1]
```diff
@@ -425,8 +425,33 @@ const DEFAULT_TYPE_CUSTOMIZATIONS: OktaApiConfig['types'] = {
     transformation: {
       fieldTypeOverrides: [
         { fieldName: 'policyRules', fieldType: 'list<PolicyRule>' },
+        { fieldName: 'settings', fieldType: 'Map<unknown>' },
```
Comment (netama) [inline packages/okta-adapter/src/config.ts:428]: nit: should be lowercase `map`

File: packages/okta-adapter/src/config.ts [hunk 2]
```diff
@@ -425,8 +425,33 @@ const DEFAULT_TYPE_CUSTOMIZATIONS: OktaApiConfig['types'] = {
...
+      fieldTypeOverrides: [
+        { fieldName: '_links', fieldType: 'list<Policy__links>' },
+      ],
+      fieldsToHide: [
+        { fieldName: 'id' },
+      ],
+      fieldsToOmit: [
+        { fieldName: 'created' },
+        { fieldName: 'lastUpdated' },
+        { fieldName: '_links' },
```
Comment (netama) [inline packages/okta-adapter/src/config.ts:453]: I thought you didn't want to omit it? (and in the place that you don't, please add a comment explaining why)

**Review by netama — APPROVED** [review]
added a couple of comments but looks good

File: packages/okta-adapter/src/config.ts (hunk 1 above)
Comment (shir-reifenberg) [reply packages/okta-adapter/src/config.ts:428]: tnx

File: packages/okta-adapter/src/config.ts (hunk 2 above)
Comment (shir-reifenberg) [reply packages/okta-adapter/src/config.ts:453]: I don't want to omit it from `Application` but from `Policy` its fine :)

**Pull request merged**